features/bevragen/persoon-beperkt/verblijfplaats/dev/fields-gba.feature

```diff
@@ -0,0 +1,394 @@
+    | 088510                  | verblijfplaats                                         | datum ingang geldigheid       |
+    | 081020                  | verblijfplaats.type                                    | gemeentedeel                  |
+    | 080910                  | verblijfplaats                                         | gemeente van inschrijving     |
+    | 081410                  | verblijfplaats.functieAdres,verblijfplaats.inOnderzoek | land vanwaar ingeschreven     |
+
+  Abstract Scenario: '<type>' van een verblijfplaats buitenland is in onderzoek en één of meerdere velden wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | land adres buitenland (13.10)   | 6014                      |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                      | waarde                       |
+    | land.code                                 | 6014                         |
+    | land.omschrijving                         | Verenigde Staten van Amerika |
+    | inOnderzoek.aanduidingGegevensInOnderzoek | <aanduiding in onderzoek>    |
+    | inOnderzoek.datumIngangOnderzoek          | 20020701                     |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | fields                                         | type                          |
+    | 080000                  | verblijfplaats                                 | hele categorie verblijfplaats |
+    | 081300                  | verblijfplaats.type                            | hele groep adres buitenland   |
+    | 081310                  | verblijfplaats                                 | land adres buitenland         |
+    | 081340                  | verblijfplaats.type                            | regel 2 adres buitenland      |
+    | 081020                  | verblijfplaats                                 | gemeentedeel                  |
+    | 080910                  | verblijfplaats                                 | gemeente van inschrijving     |
+    | 081410                  | verblijfplaats.type,verblijfplaats.inOnderzoek | land vanwaar ingeschreven     |
```

Comment:
Ik denk dat ik iets mis in deze.... Ik snap niet goed hoe ik gemeentedeel, gemeente van inschrijving en land vanwaar ingeschreven kan plaatsen in dit verband. Gaat het in dit geval niet uitsluitend om het in onderzoek zijn van de gegevens die bepalen of een een verblijfplaats is van het type verblijfplaats buitenland?